solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/properties/SolaceCommonProperties.java

```diff
@@ -8,12 +8,18 @@
 	 * This should only be set to false if you have externally pre-provisioned the required queue on the message broker.
 	 */
 	private boolean provisionDurableQueue = true;
+
 	/**
 	 * Whether to add topic subscriptions to durable queues for non-anonymous consumer groups.
 	 * This should only be set to false if you have externally pre-added the required topic subscriptions (the destination topic should be added at minimum)
 	 * on the consumer group’s queue on the message broker. This property also applies to topics added by the queueAdditionalSubscriptions property.
 	 */
+  @Deprecated
 	private boolean provisionSubscriptionsToDurableQueue = true;
```

Comment:
Should add this to the getter:
```java
@DeprecatedConfigurationProperty(reason = "Since version 2.3.0, this property is deprecated in favor of addDestinationAsSubscriptionToQueue.")
```

So that the deprecation reason is added to the configuration metadata.

Reply:
Great idea - done